apps/asap-server/src/entities/research-output.ts

```diff
@@ -49,6 +53,11 @@ export const parseGraphQLResearchOutput = (
     accessInstructions: output.flatData?.accessInstructions || undefined,
     ...optionalAuthors,
     ...optionalTeams,
+    sharingStatus: output.flatData?.sharingStatus || 'Public',
```

Comment:
```suggestion
    sharingStatus: output.flatData?.sharingStatus || 'Network Only',
```

The default is "Network Only"